types/ember__runloop/ember__runloop-tests.ts

```diff
@@ -15,9 +15,13 @@ import {
     schedule,
     scheduleOnce,
     throttle,
+    Timer,
 } from "@ember/runloop";
 import { Backburner, DebugInfo, DeferredActionQueues, QueueItem } from "@ember/runloop/-private/backburner";
 
+declare const timer: Timer;
+timer; // $ExpectType Timer
```

Comment:
It's a little funny, but something like this is more comprehensive.

```suggestion
declare const timer: Timer;
if (!((typeof timer === 'string' || typeof timer === 'number'))) {
    timer; // $ExpectType never
}
const timerString: Timer = 'abc';
const timerNumber: Timer = 123;
```